lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -479,19 +479,31 @@ class MongoClientInterface {
                 if (!doc) {
                     return cb(errors.NoSuchBucket);
                 }
+                log.info('getBucketAttributes: bucket found', {
```

Comment:
seems way too complex for logs, logs should only include what we have...
esp. on getBucketAttributes, i don't think we want to add all logs...